src/main/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsView.java

```diff
@@ -0,0 +1,543 @@
+        });
+        journalAbbreviationsTable.itemsProperty().addListener((observable, oldvalue, newvalue) -> {
+            viewModel.abbreviationsProperty().set(newvalue);
+        });
+        isEditableAndRemovable.addListener((observable, oldvalue, newvalue) -> {
+            removeJournalAbbreviationsButton.setDisable(newvalue.booleanValue());
+        });
+    }
+
+    public void showAndWait() {
+        FXAlert journalAbbreviationsDialog = new FXAlert(AlertType.INFORMATION,
+                Localization.lang("Journal abbreviations"));
+        journalAbbreviationsDialog.setResizable(true);
+        journalAbbreviationsDialog.setDialogPane((DialogPane) this.getView());
+        ((Stage) this.getView().getScene().getWindow()).setMinHeight(400);
+        ((Stage) this.getView().getScene().getWindow()).setMinWidth(600);
+        journalAbbreviationsDialog.showAndWait();
+    }
+
+    @FXML
+    private void addNewFile() {
+        FileChooser chooser = new FileChooser();
+        chooser.getExtensionFilters().add(new ExtensionFilter("TXT files (*.txt)", "*.txt"));
+        File file = chooser.showSaveDialog(null);
+        if (file != null) {
+            try {
+                viewModel.addNewFile(file.getAbsolutePath());
+                journalFilesBox.getSelectionModel().selectLast();
+            } catch (JabRefException e) {
+                showErrorDialog(e);
+            }
+        }
+    }
+
+    @FXML
+    private void openFile() {
+        FileChooser chooser = new FileChooser();
+        chooser.getExtensionFilters().add(new ExtensionFilter("TXT files (*.txt)", "*.txt"));
+        File file = chooser.showOpenDialog(null);
+        if (file != null) {
+            try {
+                viewModel.openFile(file.getAbsolutePath());
+                journalFilesBox.getSelectionModel().selectLast();
+            } catch (JabRefException e) {
+                showErrorDialog(e);
+            }
+        }
+    }
+
+    @FXML
+    private void removeList() {
+        viewModel.removeCurrentList();
+    }
+
+    @FXML
+    private void fileChanged() {
+        viewModel.changeActiveFile(journalFilesBox.getSelectionModel().getSelectedItem());
+    }
+
+    @FXML
+    private void addAbbreviation() {
+        viewModel.abbreviationsNameProperty().set("Name");
+        viewModel.abbreviationsAbbreviationProperty().set("Abbreviation");
+        try {
+            viewModel.addAbbreviation();
+        } catch (JabRefException e) {
+            showErrorDialog(e);
+        }
+        selectNewAbbreviation();
+    }
+
+    @FXML
+    private void editAbbreviation() {
+        journalAbbreviationsTable.edit(journalAbbreviationsTable.getSelectionModel().getSelectedIndex(),
+                journalTableNameColumn);
+    }
+
+    private void selectNewAbbreviation() {
+        int lastRow = viewModel.abbreviationsCountProperty().get() - 1;
+        journalAbbreviationsTable.scrollTo(lastRow);
```

Comment:
Moving the call of `scrollTo()` to the current abbreviation property listener means the view will always scroll so the selected row is the first row in the visible view of rows. This means if if I select a row in the middle of the view it will still scroll so the selected row is the first row. Therefore I would rather leave it like this.